File: UserStory/UserStory/Models/ModelTask/TaskUpdateDTO.cs
using System.ComponentModel.DataAnnotations.Schema;
using System.ComponentModel.DataAnnotations;
using UserStory.Models.ModelFunctionality;

namespace UserStory.Models.ModelTask
{
    public class TaskUpdateDTO
    {
        public Guid TaskId { get; set; }
        public string TextTask { get; set; }
        public Guid FunctionalityId { get; set; }
        public Functionality Functionallity { get; set; }

        public Guid? SprintId { get; set; }
    }
}

File: FrontendTemplateProject-main/src/app/models/UserStoryModel/taskUpdateDto.ts
import { Guid } from 'guid-typescript';
import { Functionality } from './functionality';

export class TaskUpdateDto {
    
    taskId!: Guid;
    textTask!: string;
    functionalityId!: Guid;
    functionality!: Functionality;
    sprintId: Guid;
    
    constructor(
        taskId: Guid,
        textTask: string,
        functionalityId: Guid,
        functionality: Functionality,
        sprintId: Guid
    ) {
        this.taskId = taskId;
        this.textTask = textTask;
        this.functionalityId = functionalityId;
        this.functionality = functionality;
        this.sprintId = sprintId;
    }
    
}